webui/webui.go

```diff
@@ -16,6 +17,10 @@ import (
 	"k8s.io/client-go/util/retry"
 )
 
+// assets holds our static web server assets.
+//go:embed assets/*
+var assets embed.FS
```

Comment:
❓ How does it work?

Reply:
the embedded FS is populated at compile-time, which mean that our standalone binary contains all our assets

Reply:
(stdlib doc about `embed` package: https://pkg.go.dev/embed)

Reply:
Nice.
Is is something to be aware of while developing locally ?

Reply:
I don't think so, as it occurs during compilation. It's the exact same build command.

> It should have 0 impact on your workflows

:awesome: